fastlane/lib/fastlane/swift_lane_manager.rb

```diff
@@ -216,14 +218,14 @@ def self.link_user_configs_to_project(updated_message: nil)
       return project_modified
     end
 
-    def self.start_socket_thread
+    def self.start_socket_thread(port: 2000)
```

Comment:
We can probably remove this default of `2000` since its being default in `socket_server.rb`? I would just set `port: nil`